holoviews/util/__init__.py

```diff
@@ -95,6 +95,94 @@ def __call__(self, *args, **params):
 
             self._cellmagic(args[0], args[1])
 
+
+    @classmethod
+    def option_groups(cls, obj, options=None, backend=None, clone=True, **kwargs):
```

Comment:
Didn't want to call this ``apply_option_groups`` or similar?

Reply:
Ah yes, `apply_groups` I guess!

Reply:
Fixed in fb81455, thanks.